src/runtime/virtcontainers/hypervisor_linux_arm64_test.go

```diff
@@ -25,10 +25,3 @@ func TestRunningOnVMM(t *testing.T) {
 	assert.NoError(err)
 	assert.Equal(expectedOutput, running)
 }
-
```

Comment:
Can we add any unit-test without CCA HW?

Reply:
The unit test is added: https://github.com/kata-containers/kata-containers/pull/10664/commits/b3876501e297be0f857e523c47a86f0ad8647fab